templates/component.html

```diff
@@ -4,14 +4,14 @@
 <div class="content-banner">
     <div class="content-wrapper">
         <div class="hxRow">
-            <div class="hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8 hxCol-xl-8">
-                <div class="inline-block">
-                  <h1 class="hxPageTitle inline-block">{{page.title}}</h1>
-                  <a class="hxBtn pull-right" href="https://github.com/rackerlabs/design-system/issues/new?title=Issue%20Regarding%20-%20{{page.url}}">Submit Issue</a>
+            <div class="hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg hxSpan-8-xl">
+                <div>
+                  <h1 class="inline-block">{{page.title}}</h1>
+                  <button class="hxBtn pull-right" href="https://github.com/rackerlabs/design-system/issues/new?title=Issue%20Regarding%20-%20{{page.url}}">Submit Issue</button>
```

Comment:
I don't think this is going to function any more (`href` doesn't apply to `<button>`). I think this will be fine as an `<a>` element, given that you're navigating away from the current URL.